common/transform/src/main/java/ai/h2o/mojos/deploy/common/transform/CsvToMojoFrameConverter.java

```diff
@@ -9,6 +9,7 @@
 
 /** Converts the CSV from {@link InputStream} to the {@link MojoFrame} for scoring. */
 public class CsvToMojoFrameConverter {
+  /** Convert the CSV data from the given {@link InputStream} to a {@link MojoFrame}. */
```

Comment:
I cannot find the reference but I am 99.9% sure that summaries should not be in the imperative from. So in this case it should be `Converts`.

same for `MojoScorer.score`

Reply:
Thanks, done.